src/main/java/hudson/plugins/s3/S3BucketPublisher.java

```diff
@@ -187,9 +187,15 @@ public void perform(@Nonnull Run<?, ?> run, @Nonnull FilePath ws, @Nonnull Launc
                 if (paths.isEmpty()) {
```

Comment:
What do you think - maybe it's better to `continue` in this case (after printing feedback)?